cohort-util/src/main/java/com/ibm/cohort/valueset/ValueSetUtil.java

```diff
@@ -27,6 +27,22 @@
 
 public class ValueSetUtil {
 
+	public static void validateArtifact(ValueSetArtifact valueSetArtifact){
+			if(valueSetArtifact.getUrl() == null){
+				//todo new kind of exception? Invalid Format? something specific
+				throw new IllegalArgumentException("URL must be supplied");
+			}
+			if(valueSetArtifact.getFhirResource().getId() == null || valueSetArtifact.getFhirResource().getId().equals("")){
+				throw new IllegalArgumentException("Identifier must be supplied, ensure that either the OID or the ID field is filled in");
+			}
+			if(valueSetArtifact.getFhirResource().getVersion() == null || valueSetArtifact.getFhirResource().getVersion().equals("")){
+				throw new IllegalArgumentException("Value Set Version must be supplied");
+			}
+			if(valueSetArtifact.getFhirResource().getCompose().getInclude() == null || valueSetArtifact.getFhirResource().getCompose().getInclude().size() == 0){
+				throw new IllegalArgumentException("Value set must include codes but no codes were included.");
+			}
+	}
+
```

Comment:
Because there was a lot of need for validation in POJOs, I introduce a dependency on javax.validation in my PR and rely on mostly annotations to define the validation constraints for model objects. Nothing wrong with what you've done here, but maybe another conversation to have with the wider team about validation and making sure we are consistent across code paths.